internal/dump/agentpolicies.go

```diff
@@ -0,0 +1,187 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package dump
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+
+	"github.com/elastic/elastic-package/internal/common"
+	"github.com/elastic/elastic-package/internal/kibana"
+)
+
+const AgentPoliciesDumpDir = "agent_policies"
+
+type AgentPolicyDumper struct {
+	name   string
+	client *kibana.Client
+
+	policy *AgentPolicy
+}
+
+type AgentPoliciesDumper struct {
```

Comment:
Nit. Could this dumper be generalized to dump also a single policy? If a name is given, it dumps only this one, if no name is given, it dumps all. So we wouldn't need two dumper implementations for the same thing.

Reply:
Done
Now there is just one struct `AgentPoliciesDumper` to manage both scenarios